packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -332,9 +348,15 @@ function UserMenu({
             {...sharedProps}
             active={isCloud}
             disabled={!account}
-            href={hosts.cloud}
-            description={<Description isActive={isCloud} product="cloud" />}
-            title="Cloud"
+            href={isGovernment ? 'https://cloud.mongodbgov.com' : hosts.cloud}
```

Comment:
This needs to be customizable for our dev,qa,staging fedramp environments. Should mirror what we have for `hosts.cloud` just with different top level domain.

Reply:
Good catch -- thank you